bitcoin/src/blockdata/script/push_bytes.rs

```diff
@@ -43,34 +44,38 @@ mod primitive {
     pub struct PushBytes([u8]);
 
     impl PushBytes {
-        /// Creates `&Self` without checking the length.
+        /// Creates `&PushBytes` without checking the length.
         ///
         /// # Safety
         ///
         /// The caller is responsible for checking that the length is less than the [`LIMIT`].
```

Comment:
> ``[`LIMIT`]``

Where is this item defined? Is this an already removed constant or did I miss the definition?

Reply:
I think it got removed during some refactors. It should just say 2<sup>32</sup>.